pact4s/README.md

```diff
@@ -0,0 +1,51 @@
+c# pact4s [Under construction]
+
+pact4s is used for contract testing.
+
+# Dependencies
+
+```scala
+  val pact4sDependencies = Seq(
+    pact4sScalaTest,
+    pact4sCirce,
+    http4sEmberClient,
+    http4sDsl,
+    http4sEmberServer,
+    http4sCirce,
+    circeCore,
+    typelevelCat,
+    scalaTest
+  )
+
+lazy val pact4s = project.in(file("pact4s"))
+  .settings(pact4sSettings)
+  .dependsOn(http % "test->test;compile->compile")
+```
+
+## Building and running contract tests
+Clone the repo.
+```
+$ git clone https://github.com/broadinstitute/cromwell.git 
+$ cd cromwell
+```
+
+If you are already using OpenJDK 17, run the following command. 
```

Comment:
I think this needs to be updated now that it's Java 11.

Reply:
Ah yeah thanks I thought i changed this already